### DISCUSSION TITLE
支持 jdbc 自定义参数追加吗？

### DISCUSSION THREAD

**Discussion by fuhaiq** [apache/kyuubi#3979] [Q&A] [answered]
嗨~我想为 jdbc 追加一个自定义参数，参数值是一个 uuid，客户端不需要知道。后端引擎是 spark，kyuubi 支持吗？谢谢

**Comment by yaooqinn** [comment, marked as answer]
https://kyuubi.readthedocs.io/en/latest/extensions/server/configuration.html

**Reply by fuhaiq** [reply]
谢谢
